### PR TITLE
Show local commits when no upstream is set

### PR DESCRIPTION
**Pull request by odnoletkov**
References #1963

### CODE REVIEW

File: autoload/fugitive.vim [hunk 1]
```diff
@@ -2846,16 +2846,19 @@ function! fugitive#BufReadStatus(...) abort
...
     if len(push) && !(push ==# pull && get(props, 'branch.ab') =~# '^+0 ')
-      call s:AddLogSection('Unpushed to ' . push, push, head)
+      call s:AddLogSection('Unpushed to ' . push, [push . '..' . head])
     endif
     if len(pull) && push !=# pull
-      call s:AddLogSection('Unpushed to ' . pull, pull, head)
+      call s:AddLogSection('Unpushed to ' . pull, [pull . '..' . head])
+    endif
+    if len(pull) == 0
+      call s:AddLogSection('Unpushed to refs/remotes', [head, '--not', '--remotes'])
```
[the remark this answers is not in the document]
Comment (tpope) [reply autoload/fugitive.vim:2855]: P.S. use `empty()`.

[the remark this answers is not in the document]
Comment (odnoletkov) [reply autoload/fugitive.vim]: Done.

Should `-` do something in this section? In a new repo, something like `Git push --set-upstream $remote $commit:$branch` could be useful. But `$remote` is ambiguous.

**Comment by odnoletkov** [comment]
@tpope should we also hide this new section during rebase? Currently displayed under rebase todo

**Comment by tpope** [comment]
Going with a tweaked version that doesn't activate during rebase. I'll save that for another day.

**Pull request closed without merging**